clerk/clerk.go

```diff
@@ -29,6 +29,7 @@ const (
 	OrganizationsUrl  = "organizations"
 	PhoneNumbersURL   = "phone_numbers"
 	RedirectURLsUrl   = "redirect_urls"
+	samlConnectionUrl = "saml_connections"
```

Comment:
For consistency with the other names, shouldn't this be:

- exported
- plural

i.e. `SAMLConnectionsUrl`

Reply:
You are right, let's be consistent. Updated!